Tabby/app/recommendations/index.tsx

```diff
@@ -27,7 +27,7 @@ const initialBooks: Book[] = [
     },
     {
         id: '2',
-        title: 'To Kill a Mockingbird',
+        title: 'To Kill',
```

Comment:
I get what you're doing here, changing the names so the search bar can be more properly tested.

But I feel keeping the names this similar may be hurting that. I think the changes should be more substantial and you can just different book names.

Some examples could be:
- Lord of the Rings
- Game of Thrones
- Twilight
- The Bible

Just take a look at these series: https://www.forbes.com/sites/entertainment/article/best-book-series/
If  you want more exact book names: https://www.barnesandnoble.com/b/books/_/N-1fZ29Z8q8

The only thing that really needs to change is the title. The description and author can stay the same unless you plan to add a filter for the author or make it to where you can search by author.